plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -42,7 +42,7 @@ export const createPublishGitlabMergeRequestAction = (options: {
     targetPath?: string;
     token?: string;
     commitAction?: 'create' | 'delete' | 'update';
-    /** @deprecated Use projectPath instead */
+    /** @deprecated projectID passed as query parameters in the repoUrl */
```

Comment:
my question remains here, should we undeprecate these in case it still make sense to have it? @Borchies and @benjdlambert

Reply:
Sorry I'm still now clear on this one. Should these deprecations be removed now that projectID are in use again or is the case that this _is deprecated_ and that it should/is passed from the repoUrl instead? :)